debugger/src/cell_renderers/cellrendererframeicon.c

```diff
@@ -223,12 +227,15 @@ static void cell_renderer_frame_icon_render(GtkCellRenderer *cell, GdkDrawable *
 static void cell_renderer_frame_icon_init (CellRendererFrameIcon *cell)
 {
 	GtkCellRenderer *cell_renderer = (GtkCellRenderer*)cell;
+	GValue mode = G_VALUE_INIT;
 	
 	cell->active_frame = FALSE;
-	
-	cell_renderer->mode = GTK_CELL_RENDERER_MODE_ACTIVATABLE;
-
 	cell->pixbuf_active = cell->pixbuf_highlighted = 0;
+
+	g_value_init(&mode, G_TYPE_ENUM);
```

Comment:
Again this should be:

    g_value_init(&mode, GTK_TYPE_CELL_RENDERER_MODE);

I found this through testing this PR. Without doing the above you cannot change the current active frame in the stack.